sapi/litespeed/lsapilib.h

```diff
@@ -417,12 +418,7 @@ int LSAPI_Postfork_Parent(LSAPI_Request * pReq);
 
 #define LSAPI_LOG_PID            (0x10000)
 
-void LSAPI_Log(int flag, const char * fmt, ...)
-#if __GNUC__
-        __attribute__((format(printf, 2, 3)))
-#endif
-;
-
+void LSAPI_Log(int flag, const char * fmt, ...) ZEND_ATTRIBUTE_FORMAT(printf, 2, 3);
```

Comment:
This seems to be isolated from Zend, I think I prefer what you had before so we don't have unnecessary dependency on Zend if not needed.

Reply:
Sure, added back as it was with only ifdef changed.

I was already thinking about these __attribute__ items and how to improve it. It might fit a bit better together once we add a minimum compiler requirement in the future to rely on __has_attribute feature that comes with GCC 5 and later and clang 2.9 and later (I think). With some other adjustments for some almost obsolete platforms. Because now PHP doesn't have this resolved optimally yet.